README.md

```diff
@@ -21,6 +21,8 @@ const apos = require('apostrophe')({
 });
 ```
 
+**Set the `baseUrl` value.** Open Graph images *will not be set* with absolute URLs if the `baseUrl` is not set. This should either be [on the core server object](https://docs.apostrophecms.org/reference/core-server.html#options) or, more likely, in environment configuration, [such as in `data/local.js`](https://docs.apostrophecms.org/core-concepts/global-settings/settings.html#hardcoded-settings).
```

Comment:
"Facebook considers an absolute URL to be a requirement and will not accept the image without it."